py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/s3_utils.py

```diff
@@ -29,3 +29,10 @@ def file_list_from_s3(bucket_name: str,
             continue
         for obj in page['Contents']:
             yield (obj['Key'], obj['Size'])
+
+def invoke_async_lambda(function_arn: str, event: dict) -> None:
+    lambda_client = boto3.client('lambda')
+    logging.info("Invoking Lambda: %s" % function_arn)
+    lambda_client.invoke(FunctionName=function_arn,
+                         InvocationType='Event',
+                         Payload=json.dumps(event))
```

Comment:
not sure how to stub this, but it follows the patterns i saw in documentation and examples.

the `InvocationType='Event'` makes this async.

Reply:
Do we want to do anything with the return object from `.invoke`? 

It offers a http `StatusCode` and `FunctionError` string that we might want to do something with.

Not sure if an invocation failure would be caught / reported anywhere else.

Reply:
if we're waiting for a response, then we can run into cases where the batch lambda is timing out because its waiting for responses from the ingest lambda. it is a good point though that this is an error who's details we'll want to capture and log. i made a ticket to do some digging into that.

https://app.asana.com/0/1189492770004753/1202389697849560/f